### DISCUSSION TITLE
Env variables in remote configuration files

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#101] [General]
Hi @neonexus,

In the `config`, we have our environment variables in the subfolder `env` as per the NODE_ENV. They are all created and used for our convenience right! They aren't looked up default by Sails unless we ask it to.

Webpack uses these variables to builds the assets for prod. No doubt in that.

But when we lift our Sails using `npm run lift:prod`, does [config/env/production.js](https://github.com/neonexus/sails-react-bootstrap-webpack/blob/release/config/env/production.js) get used by Sails. It isn't right?
How do we ask Sails to use it while lifting?

Thanks

**Comment by neonexus** [comment]
`npm run lift:prod` should indeed use the `production` config; it is "the same thing as `sails lift --prod` or `NODE_ENV=production node app.js`".

If you have a `config/local.js`, that overrides everything; including the production configuration.

**Reply by MaheshkumarSundaram** [reply]
Ah ok. Thanks
